v0/destinations/revenue_cat/data/RCIdentifyConfig.json

```diff
@@ -0,0 +1,151 @@
+  },
+  {
+    "destKey": "$phoneNumber.value",
+    "sourceKeys": "phone",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "$apnsTokens.value",
+    "sourceKeys": "traits.apnsTokens",
+    "required": false
+  },
+  {
+    "destKey": "$fcmTokens.value",
+    "sourceKeys": "traits.fcmTokens",
+    "required": false
+  },
+  {
+    "destKey": "$idfa.value",
+    "sourceKeys": ["context.idfa", "context.device.advertisingId"],
+    "required": false
+  },
+  {
+    "destKey": "$idfv.value",
+    "sourceKeys": ["context.idfv", "context.device.id"],
+    "required": false
+  },
+  {
+    "destKey": "$gpsAdId.value",
+    "sourceKeys": "traits.gpsAdId",
+    "required": false
+  },
+  {
+    "destKey": "$androidId.value",
+    "sourceKeys": "context.device.advertisingId",
+    "required": false
+  },
+  {
+    "destKey": "$campaign.value",
+    "sourceKeys": "context.campaign.name",
+    "required": false
+  },
+  {
+    "destKey": "$ip.value",
+    "sourceKeys": ["context.ip", "request_ip"],
+    "required": false
+  },
+  {
+    "destKey": "$creative.value",
+    "sourceKeys": "traits.creative",
+    "required": false
+  },
+  {
+    "destKey": "$keyword.value",
+    "sourceKeys": "traits.keyword",
+    "required": false
+  },
+  {
+    "destKey": "$mediaSource.value",
+    "sourceKeys": "traits.mediaSource",
+    "required": false
+  },
+  {
+    "destKey": "$ad.value",
+    "sourceKeys": "traits.ad",
+    "required": false
+  },
+  {
+    "destKey": "$adGroup.value",
+    "sourceKeys": "traits.addGroup",
+    "required": false
+  },
+  {
+    "destKey": "$adjustId.value",
+    "sourceKeys": "traits.adjustId",
+    "required": false
+  },
+  {
+    "destKey": "$appsflyerId.value",
+    "sourceKeys": "traits.appsflyerId",
+    "required": false
+  },
+  {
+    "destKey": "$fbAnonId.value",
+    "sourceKeys": "traits.fbAnonId",
+    "required": false
+  },
+  {
+    "destKey": "$iterableUserId.value",
+    "sourceKeys": "traits.iterableUserId",
+    "required": false
+  },
+  {
+    "destKey": "$iterableCampaignId.value",
+    "sourceKeys": "traits.iterableCampaignId",
+    "required": false
+  },
+  {
+    "destKey": "$iterableTemplateId.value",
+    "sourceKeys": "traits.iterableTemplateId",
+    "required": false
+  },
+  {
+    "destKey": "$mixpanelDistinctId.value",
+    "sourceKeys": "traits.mixpanelDistinctId",
+    "required": false
+  },
+  {
+    "destKey": "$mparticleId.value",
+    "sourceKeys": "traits.mparticleId",
+    "required": false
+  },
+  {
+    "destKey": "$onesignalId.value",
+    "sourceKeys": "traits.onesignalId",
+    "required": false
+  },
+  {
+    "destKey": "$airshipChannelId.value",
+    "sourceKeys": "traits.airshipChannelId",
+    "required": false
+  },
+  {
+    "destKey": "$amazonAdId.value",
+    "sourceKeys": "traits.amazonAdId",
+    "required": false
+  },
+  {
+    "destKey": "$clevertapId.value",
+    "sourceKeys": "traits.clevertapId",
+    "required": false
+  }
+]
```

Comment:
As this config is for identify event call, add `context.traits.xxx` wherever you have used `traits.xxx`.

Reply:
Yeah done